### DISCUSSION TITLE
Get some warnings then execute ./install.sh script

### DISCUSSION THREAD

**Discussion by TAI-REx** [amidaware/tacticalrmm#1074] [General]
**Server Info (please complete the following information):**
 - OS: [Debian 10]
 - Browser: -
 - RMM Version: v0.12.4

**Installation Method:**
  - [*] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
get some warnings then execute ./install.sh script 

1. some warnings then execute ./install.sh
 > gcc -pthread -fPIC -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fno-semantic-interposition -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-missing-field-initializers -Werror=implicit-function-declaration -fvisibility=hidden -fprofile-generate -I./Include/internal -I./Include -I. -I/usr/include/x86_64-linux-gnu -I/usr/local/include -I/home/tactical/Python-3.10.2/Include -I/home/tactical/Python-3.10.2 -c /home/tactical/Python-3.10.2/Modules/socketmodule.c -o build/temp.linux-x86_64-3.10/home/tactical/Python-3.10.2/Modules/socketmodule.o
/home/tactical/Python-3.10.2/Modules/socketmodule.c: In function ‘getsockaddrarg’:
/home/tactical/Python-3.10.2/Modules/socketmodule.c:2350:9: warning: ‘strncpy’ specified bound 64 equals destination size [-Wstringop-truncation]
 2350 |         strncpy((char *)sa->salg_name, name, sizeof(sa->salg_name));
      |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

2. after sqanning a QR code for 2fa verif get some No such file or directory error
> chown: cannot access '/home/tactical/.config': No such file or directory

3.also need to add to manual that need to ssh login via domain name coz have a lot of issues like:
> sudo: unable to resolve host [hostname]: Name or service not known

**To Reproduce**
1. exec. ./install.sh on Debian 10

**Expected behavior**
-

**Screenshots**
-

**Additional context**
All deployment actions was done via manual
>> https://docs.tacticalrmm.com/install_server/

**Comment by dinger1986** [comment]
Has this stopped it working?

**Comment by silversword411** [comment]
You sure this isn't a bad install. New vm, reinstall: 
https://docs.tacticalrmm.com/troubleshooting/#problems-after-new-server-install

For troubleshooting you install it's best to use discord.

**Comment by dinger1986** [comment]
All look like normal errors to me, so assume it's working, can leave open for afew days then close?
